SU-HKKU/cardinal_service_profile
# Fixed icon after the link on resources for firefox

# READY FOR REVIEW

# Summary
- Fixed the position of the link icon for resources.

# Need Review By (Date)
- :shrug: 

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. checkout the latest branch of the gryphon stack `main` branch.
3. sync the site `blt drupal:sync --site=cardinalservice`
4. view the `/resources` page in firefox
5. verify the right pointing icon is in the correct position
6. verify the position in other browsers as well.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
